README.md

```diff
@@ -18,7 +18,10 @@ Joinmarket is coinjoin software, and includes a wallet, which requires Bitcoin C
 
 ### Quickstart
 
-Once you've downloaded this repo, either as a zip file, and extracted it, or via `git clone`:
+Once you've downloaded this repo, either as a tar/zip file, and extracted it, or via `git clone`:
+
+Make sure to validate the signature on the tar/zip file provided on the [https://github.com/Joinmarket-Org/joinmarket-clientserver/releases](release page),
+or check the signature in git if you install that way using `git log --show-signature`.
```

Comment:
A note on how to get the gpg keys and their fingerprints may be useful here.

Reply:
Yes, a couple of people said the same thing on IRC, but if they go to that releases page directly linked there, it tells them about the key, but I agree a sentence telling them how/where to get the info would be desirable, perhaps even some instructions? Could you or someone else suggest a sentence here with a link if appropriate?